rebasehelper/tests/testing_files/test.spec

```diff
@@ -75,6 +75,10 @@ make check
 
 %files
 %{_bindir}/file.txt
+/usr/share/man/man1/*
+/usr/share/test1.txt
+/usr/bin/test2.txt
+/dev/test3
```

Comment:
I think we should try to keep the testing spec file sane. It doesn't make sense to install a `txt` file to `/usr/bin`, nor installing anything to `/dev`. Since it's probably impossible to find a real path that wouldn't be substituted with any macro, I'd use something like `/no/macros/here`.

You should also add something like `%config(noreplace) /etc/test/test.conf`.